rbac/management/group/definer.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Handler for system defined group."""
+import logging
+
+from management.group.model import Group
+from management.policy.model import Policy
+from management.principal.model import Principal
+from management.role.model import Role
+from tenant_schemas.utils import tenant_context
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def _create_group(tenant, role_list):
+    """Create the group object in the database."""
+    with tenant_context(tenant):
+        name = "Platform group"
+        group_description = "Default group that contains default permissions for new users."
+        policy_description = "Platform default policy that is binded with default group."
+
+        principals = list(Principal.objects.all())
+        group = Group.objects.create(name=name,
+                                     description=group_description,
+                                     system=True)
+        group.principals.add(*principals)
+        group.save()
+
+        policy = Policy.objects.create(name="Platform policy",
+                                       description=policy_description,
+                                       system=True,
+                                       group=group)
+        policy.roles.add(*role_list)
+        policy.save()
+        logger.info('Created group %s for tenant %s.', name, tenant.schema_name)
+
+    return group
+
+
+def seed_group(tenant):
+    """For a tenant create default group."""
+    platform_role_list = list(Role.objects.filter(platform=True, system=True))
```

Comment:
I think the other flag is going to be "platform_default"? @wcmitchell unless you were going to go with just "platform"?

Reply:
Looks like @wcmitchell has it set here: https://github.com/RedHatInsights/insights-rbac/pull/151 😄

Reply:
Right, I would rebase my PR according to his.